Tests/backend-specific-tests/Synapse/RunnerSynapseTest.php

```diff
@@ -479,4 +482,86 @@ public function testAbsWorkspaceMappingFilesOutput()
         $options->setConfigurationId($configId);
         self::assertCount(0, $components->listConfigurationWorkspaces($options));
     }
+
+    public function testAbsWorkspaceOutputTablesAsFiles()
+    {
+        if (!RUN_SYNAPSE_TESTS) {
+            self::markTestSkipped('Synapse test is disabled.');
```

Comment:
this could be in the setup method, so it doesn't have to repeat in every test :)